x-pack/plugins/maps/public/licensed_features.ts

```diff
@@ -33,6 +34,10 @@ export const LICENCED_FEATURES_DETAILS: Record<LICENSED_FEATURES, LicensedFeatur
     name: 'layer from local Elastic Maps Server',
     license: 'enterprise',
   },
+  [LICENSED_FEATURES.ML_ANOMALIES]: {
```

Comment:
discussed this offline with @nreese 

I originally introduces this for completeness, but having these licensed_features-types extended is not strictly necessary. It's solely used for some additional telemetry.

Since it is not critical, we don't think it is necessary to provide API around it to make this extensible

So we can remove this.